backend/api/views/dataset.py

```diff
@@ -84,4 +86,17 @@ def datawork_import(self, request):
         queryset = new_datasets.annotate(Count("files")).select_related("dataset_type")
         serializer = self.serializer_class(queryset, many=True)
 
+        try:
+            check_new_spectro_config()
+        except FileNotFoundError as error:
+            regex = "dataset/(.*)/analysis"
+            buggy_dataset = re.findall(regex, str(error))
+            check_error = (
+                f"Successful import. Reload (F5) this page to see it<br/>"
+                f"But an another dataset config spectro ({buggy_dataset[0]}) can't be update : <br/>"
+                f"{error}"
```

Comment:
Don't use html in an error code please. Let's keep it as json only and then in the front do what you want to "upgrade" the error message with html :wink:.

Reply:
What is the best method to do that ? Or if I want to put a new line, I could use "\n" it's stay string :satisfied:

Reply:
Maybe return something like `{"error_lines": ["line 1", "line 2"]]}` then in JS you do the equivalent of `"<br/>".join(result["error_lines"]`

Reply:
Why did you use `\n` instead of what I suggested ?